packages/react-components/src/components/styles/ImageGallery.style.ts

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IIconProps, mergeStyles, Theme } from '@fluentui/react';
+
+/**
+ * @private
+ */
+export const cancelIcon: IIconProps = { iconName: 'Cancel' };
+
+/**
+ * @private
+ */
+export const downloadIcon: IIconProps = {
+  iconName: 'Download'
+};
+
+/**
+ * @private
+ */
+export const overlayStyles = {
+  root: { background: 'rgba(0, 0, 0, 0.85)' }
```

Comment:
nit: I would do this with the `opacity` CSS property, if you need the colour lets try to do that with the theme like you do below.